src/components/views/society/Society.js

```diff
@@ -1,11 +1,69 @@
 import React from 'react';
 import Navbar from '../../navigation/Navbar';
+import Card from './Card';
+import abadina from './assets/Ellipse 3.png';
+import community from './assets/Ellipse 4.png';
+import international from './assets/Ellipse 5.png';
+import styles from './Society.module.css';
 
-function Society() {
+function Society({ image, name, link }) {
   return (
     <>
       <Navbar />
-      <div>Society</div>
+      <div className={styles.header}>
+        <h1>MSSNAAC Societies</h1>
+        <p>Our diverse range of student societies </p>
+      </div>
+      <div className={styles.cards}>
+        <Card
+          className={styles.card}
+          image={abadina}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+        <Card
+          className={styles.card}
+          image={community}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+        <Card
+          className={styles.card}
+          image={international}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+      </div>
+      <div className={styles.cards}>
+        <Card
+          className={styles.card}
+          image={abadina}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+        <Card
+          className={styles.card}
+          image={community}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+        <Card
+          className={styles.card}
+          image={international}
+          name="Abadina College
+          UI, Ibadan"
+          link="website"
+        />
+      </div>
+      <div className={styles.btn}>
+        <button type="button" className={styles.prev}>Prev</button>
+        <button type="button" className={styles.next}>Next</button>
+      </div>
```

Comment:
It would be nice if we enable the `Prev` and `Next` buttons to navigate through pages, we have more than 50 schools to include in that category.

So implementing that will be a nice feature 💯💪

![image](https://github.com/abdulhamiid/mssnaac/assets/78251927/4e01f270-0dc2-410b-bfe0-9f745258a6c1)

Also, kindly update the `font-family` of the button and the `text-color` the match the design on Figma 🛠🤗👍

Figma 👀
![image](https://github.com/abdulhamiid/mssnaac/assets/78251927/ec67f561-a49b-4803-b019-baaba7ff3dda)